tests/cypress/tests-e2e/layers.cy.js

```diff
@@ -885,9 +885,9 @@ describe('Test of layer handling', () => {
                     .should('be.visible')
                     .click()
                 // change the opacity to check later on that the new layer as the non default opacity
-                cy.get(`[data-cy="slider-opacity-layer-${timedLayerId}-2"]`)
+                cy.get(`[data-cy="slider-transparency-layer-${timedLayerId}-2"]`)
                     .should('be.visible')
-                    .realClick()
+                    .realClick({ position: 'right', force: true })
```

Comment:
Force true is not a good idea why do you need it ?

Reply:
that's a leftover from when I was testing. I thought that maybe something was blocking the click. I'm removing it right now.